.github/workflows/pull_requests/need_2nd_review.yml

```diff
@@ -0,0 +1,45 @@
+name: Need second review
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  approved:
+    if: github.event.review.state == 'approved'
+    runs-on: ubuntu-latest
+    steps:
+    -  uses: tspascoal/get-user-teams-membership@v2
+       id: checkUserMember
+       with:
+         organization: PrestaShop
+         username: ${{ github.actor }}
+         team: 'committers'
+         GITHUB_TOKEN: ${{ secrets.GHPROJECT_TOKEN }}
+    - if: ${{ steps.checkUserMember.outputs.isTeamMember == 'true' }}
+      run: echo "yes"
+
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: '8.1'
+        tools: composer
+
+    - uses: actions/checkout@v2
+      with:
+        name: prestashop/kanbanbot
+
+    - name: Install dependencies
+      run: |
+        composer self-update --2.2
+        composer install --no-interaction
+
+    - name: Test PHP
+      env:
+        GITHUB_TOKEN: ${{ secrets.GHPROJECT_TOKEN }}
+        ORGANIZATION: PrestaShop
+        PROJECT: PrestaShop
+        PROJECT_NUMBER: 17
```

Comment:
these 3 hardcoded variables need to be moved. But we want to go forward and use the bot. So this (needed) change will be an improvement later.